build.cake

```diff
@@ -123,8 +123,8 @@ Task("GenerateEvents")
     .IsDependentOn("BuildEventBuilder")
     .Does (() =>
 {
-    var eventBuilder = "./src/EventBuilder/bin/Release/EventBuilder.exe";
-    var workingDirectory = "./src/EventBuilder/bin/Release";
+    var eventBuilder = "./src/EventBuilder/bin/Release/net45/EventBuilder.exe";
+    var workingDirectory = "./src/EventBuilder/bin/Release/Net45";
```

Comment:
Given the build failing on AppVeyor, would it be worth us dumping some of the logging output to the console around here to make it easier to debug?